src/simulation/MixingModels.hh

```diff
@@ -494,9 +495,21 @@ void DiffusionMixingModel<T>::generateInflows(T timeStep, arch::Network<T>* netw
                     this->injectMixtureInEdge(newMixture->getId(), channelId);
                 }
                 // TODO add how the concentration is then stored again
+                else if (cfdSimulator->getFlowRates().at(nodeId) < 0.0 && (channel->getNodeA() == nodeId || channel->getNodeB() == nodeId)) {
+                    for (const auto& [specieId, concentrationField] : concentrationFieldsOut.at(nodeId)) {
+                        // if (!outflowDistributions.at(channel->getId()).empty()) {
+                        //     auto& section = outflowDistributions.at(channel->getId()).front(); // there is only one channel connected to a node at a CFD hybrid node connection
+                        //     // std::vector<T> inflowSectionIntoCfd = section.concentrationField;
+                        // } 
+                        int resolutionIntoCfd = 20; // TODO this should be a parameter
```

Comment:
Here the lattice resolution is hard coded rather than passed from the simulator.

Reply:
The resolution values are stored in olbContinuous.h or essContinuous.h and can technically be obtained from a cfdSimulator with `cfdSimulator->getResolution()`. This is not yet implemented. I will take care of this.

**TODO**

Reply:
Okay, thank you, please let me know if/when I need to adapt something here as well